doc/usage/restructuredtext/field-lists.rst

```diff
@@ -9,19 +9,60 @@ fields marked up like this::
 
     :fieldname: Field content
 
-Sphinx provides custom behavior for bibliographic fields compared to docutils.
+Field lists are :duref:`originally defined in docutils <field-lists>`
+to show information about a page (such as the document author or date of
+publication).
+
+.. note::
+
+    The *values* of field lists will be parsed as
+    strings, you cannot use Python collections such as lists or dictionaries.
+
+Sphinx treats field lists slightly differently, as explained
+below.
 
 .. _metadata:
 
 File-wide metadata
 ------------------
 
 A field list near the top of a file is normally parsed by docutils as the
-*docinfo* which is generally used to record the author, date of publication and
-other metadata.  However, in Sphinx, a field list preceding any other markup is
-moved from the *docinfo* to the Sphinx environment as document metadata and is
-not displayed in the output; a field list appearing after the document title
-will be part of the *docinfo* as normal and will be displayed in the output.
+*docinfo* and shown on the page.  However, in Sphinx, a field list preceding
+any other markup is moved from the *docinfo* to the Sphinx environment as
+document metadata, and is not displayed in the output.
+
+.. note::
+
+    A field list appearing after the document title *will* be part of the
+    *docinfo* as normal and will be displayed in the output.
+
+Any key / value pairs in field lists that are placed *before* the page's
+title will be available to the Jinja template when building the page within
+the :data:`meta` attribute. For example, if a page had the following text
+before its first title:
```

Comment:
It would be better to separate mark-ups and how-to write themes. This might confuse the readers.

Reply:
Where do you think would be a better place for this information? And is there any other case that a user would use field lists other than for their use within a theme?

Reply:
I think this would be best placed in the HTML themes development document.